src/cpp/cpp_declarator_converter.cpp

```diff
@@ -11,6 +11,12 @@ Author: Daniel Kroening, kroening@cs.cmu.edu
 
 #include "cpp_declarator_converter.h"
 
+//#define DEBUG
+
+#ifdef DEBUG
```

Comment:
Possibly this is just ending up in the wrong commit - but we don't seem to use `iostream` in this cpp. ( @peterschrammel )